src/cdb.c

```diff
@@ -101,7 +101,7 @@ bool cdb_read(struct cdb *c, char *buf, ut32 len, ut32 pos) {
 		return false;
 	}
 	while (len > 0) {
-		ssize_t r = read (c->fd, buf, len);
+		int r = read (c->fd, buf, len);
```

Comment:
i think that if we cast to int it will be good on all platforms: `int r = (int)read ..`